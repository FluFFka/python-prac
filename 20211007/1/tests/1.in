(1, 1), (2, 3), (51, 1), (5, 3), (2, 2), (33,3)
