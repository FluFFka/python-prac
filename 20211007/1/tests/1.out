((51, 1), (33, 3))
